NSS-Day-Cohort-49/giffygram-success-kids
# Ar starting giffygram

#### Changes Made
1. Made post entry form. 
1. Form inputs and buttons are functioning properly.
​
#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout branchname
    ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
    > Instructions for how reviewer can test functionality, and detailed description of what the expected outcome is.
    > Example: When user fills out the form and hits save, then their gif should post. Or if they hit cancel, it should clear all fields they've filled out previously. 
4. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.
